inspectors/view/qml/notation/mmrests/MMRestPopup.qml

```diff
@@ -0,0 +1,83 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.2
+import MuseScore.Inspectors 3.3
+import MuseScore.UiComponents 1.0
+import MuseScore.Ui 1.0
+import "../../common"
+
+StyledPopup {
+    id: root
+
+    property QtObject model: null
+
+    implicitHeight: contentColumn.implicitHeight + topPadding + bottomPadding
+    width: parent.width
+
+    Column {
+        id: contentColumn
+
+        width: parent.width
+
+        spacing: 12
+
+        Column {
+            spacing: 8
+
+            height: childrenRect.height
+            width: parent.width
+
+            Item {
+                height: childrenRect.height
+                width: parent.width
+
+                Column {
+                    anchors.left: parent.left
+                    anchors.right: parent.horizontalCenter
+                    anchors.rightMargin: 2
+
+                    spacing: 8
+
+                    CheckBox {
+                        id: numberVisibilityCheckBox
+
+                        isIndeterminate: root.model ? root.model.isNumberVisible.isUndefined : false
+                        checked: root.model && !isIndeterminate ? root.model.isNumberVisible.value : false
+                        text: qsTr("Number visible")
+
+                        onClicked: { root.model.isNumberVisible.value = !checked }
+                    }
+                }
+
+                Column {
```

Comment:
It's better to use InspectorPropertyControl, so users will be able to reset your property value to defaults

Reply:
Will do.